Taskfile.yml

```diff
@@ -30,7 +30,6 @@ tasks:
       - task compose-up
       - task composer-install
       - task db:migrate --yes
```

Comment:
Can we do an "if fixtures apply fixtures"?

Reply:
Added extra command for this